integration-tests/tests/src/index.ts

```diff
@@ -32,23 +28,28 @@ if (!global.path) {
     throw new Error("Expected 'path' to be available as a global");
 }
 
+if (!global.fetch) {
+    throw new Error("Expected 'fetch' to be available as a global");
+}
+
 if (!global.environment || typeof global.environment !== "object") {
     throw new Error("Expected 'environment' to be available as a global");
 }
 
 // Patch in a function that can skip running tests in specific environments
-import { skipIf } from "./utils/skip-if";
-global.it.skipIf = skipIf;
-
-describe(global.title, () => {
-    require("./realm-constructor");
-    require("./serialization");
-    require("./objects");
-    require("./iterators");
-    require("./dynamic-schema-updates");
-    require("./bson");
-});
+import { testSkipIf, suiteSkipIf } from "./utils/skip-if";
+global.describe.skipIf = suiteSkipIf;
+global.it.skipIf = testSkipIf;
```

Comment:
We should be less magical here and take an expression instead of a string, list of strings or object:

    it.skipIf(environment.node, "works", () => {});